_templates/download.html

```diff
@@ -21,7 +21,9 @@
 <link rel="alternate" type="application/rss+xml" href="/en/rss/releases.rss" title="Bitcoin Core releases">
 <div class="download">
   <h1>{% translate pagetitle %}</h1>
-  <h2>{% translate latestversion %} {{ site.DOWNLOAD_VERSION }}<a type="application/rss+xml" href="/en/rss/releases.rss"><img src="/img/icons/icon_rss.svg" alt="rss" class="rssicon"></a></h2>
+  <div class="inline-block">
+    <h2 id="latestversion">{% translate latestversion %} {{ site.DOWNLOAD_VERSION }}<a type="application/rss+xml" href="/en/rss/releases.rss"><img src="/img/icons/icon_rss.svg" alt="rss" class="rssicon"></a></h2>
+  </div>
```

Comment:
@saivann thanks for catching that!  I really do have to remember to do more mobile testing.  I fixed it by removing the commit that added the workaround, removing the header IDs so no anchor links are displayed, and exempting the Download page from the anchor link checking (it's not like anyone is likely to link to the anchors on this page anyway).  I checked that in Firefox's responsive test mode and it's no longer broken.